src/modules.js/saveDb.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable  */
+
+import { showDb } from './showDb.js';
+import { changeIcons } from './changeIcons.js';
+
+export const saveDb = (arryOfActivities) => {
+  localStorage.setItem('to-do-list', JSON.stringify(arryOfActivities));
+  showDb(arryOfActivities);
+  changeIcons();
+};
```

Comment:
- [ ] there's a small bug when i delete an item and then add a new one it will get a repetitive index (same as the last item added to the list ) , but it should have an unique index as mentioned in the requirements , kindly fix that 🙏 

![reptitivieeee](https://user-images.githubusercontent.com/89282221/235841161-685b751c-e9f9-4ead-8e00-1f3d553b9591.gif)

Reply:
fixed